src/UCI.cpp

```diff
@@ -265,10 +264,10 @@ uint64_t UCI::perft(BoardHistory& bh, Depth depth) {
 /// In addition to the UCI ones, also some additional debug commands are supported.
 
 void UCI::loop(const std::string& start) {
-
   string token, cmd = start;
   BoardHistory bh;
   bh.set(Position::StartFEN);
+  static auto search = std::make_unique<UCTSearch>(bh.shallow_clone());
```

Comment:
Why is this static?

Reply:
Ah right. It's required for the way LZGo GTP::execute works but not for LZChess UCI::loop. Will fix this.